interface/reports/clinical_stats_by_demographics_report.php

```diff
@@ -155,9 +125,14 @@ function show_all_diags(){
             type: "POST",
             url: "../../library/ajax/clinical_stats_and_lab_stats_by_demographics_report_ajax.php",
             data: {
+
                 func:"get_all_diags_data",
-                diag:"<?php  echo $_POST['form_diagnosis']   ?>",
-                proc:"<?php  echo $_POST['form_service_codes']   ?>"
+                diag:"<?php  echo $_POST['form_diagnosis'];   ?>",
+                proc:"<?php  echo $_POST['form_service_codes'];   ?>",
```

Comment:
Why is this field in this report, it's not used here

Reply:
specifically "form_service_codes"

Reply:
Removed it